README.md

```diff
@@ -6,12 +6,17 @@ A service for allocating Prisoners to Prisoner Offender Managers (POMs).
 
 ## Technical Information
 
-This is a Ruby on Rails application that exposes an interface for Prison staff
-to manage the allocation of POMs to Prisoners.
+This Ruby on Rails application enables prison staff to manage the allocation of POMs to prisoners.
 
-[![auto-updating container diagram](https://static.structurizr.com/workspace/56937/diagrams/omic-container.png)](https://structurizr.com/share/56937/diagrams#omic-container)
+### API Specification
 
-👆 edit in [hmpps-architecture-as-code](https://github.com/ministryofjustice/hmpps-architecture-as-code/blob/9990e7fbb3aa545208d2ebc40104f6f3d5a9813d/src/main/kotlin/model/OffenderManagementInCustody.kt)
+[![API docs](https://img.shields.io/badge/API_docs-view-85EA2D.svg?logo=swagger)](https://allocation-manager-staging.apps.live-1.cloud-platform.service.justice.gov.uk/api-docs/index.html)
+
+### Architecture - Context View
+
+[![auto-updating container diagram](https://static.structurizr.com/workspace/56937/diagrams/manage-POM-cases-container.png)](https://structurizr.com/share/56937/diagrams#manage-POM-cases-container)
+
+👆 edit in [hmpps-architecture-as-code](https://github.com/ministryofjustice/hmpps-architecture-as-code/blob/9990e7fbb3aa545208d2ebc40104f6f3d5a9813d/src/main/kotlin/model/manage-pom-cases.kt)
```

Comment:
@rj-adams it looks like this diagram image and the links are broken. I can see you've changed them to reference `manage-pom-cases` now instead of `omic-container`. Have you changed the files in the `hmpps-architecture-as-code` repository to reflect the new name?

e.g. this old link works:
https://structurizr.com/share/56937/diagrams#omic-container

but the new link fails with "view not found":
https://structurizr.com/share/56937/diagrams#manage-POM-cases-container